getInfo/getInfo.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+function getInfo ($all)
```

Comment:
get what info? What is $all? Can you:
1) Give this function a better name
2) Give all a better name
3) Add type hints

Reply:
UNIT TEST ME PLEASSSSEEEEE